auditlog_tests/tests.py

```diff
@@ -81,6 +81,49 @@ def test_update(self):
             msg="The change is correctly logged",
         )
 
+    def test_update_specific_field_supplied_via_save_method(self):
+        obj = self.obj
+
+        # Change 2 fields, but save one only.
+        obj.boolean = True
+        obj.text = "Short text"
+        obj.save(update_fields=["boolean"])
+
+        # This implicitly asserts there is only one UPDATE change since the `.get` would fail otherwise.
+        self.assertJSONEqual(
+            obj.history.get(action=LogEntry.Action.UPDATE).changes,
+            '{"boolean": ["False", "True"]}',
+            msg="Object modifications that are not saved to DB are not logged when using the `update_fields`.",
+        )
+
+    def test_django_update_fields_edge_cases(self):
+        """
+        The test ensures that if Django's `update_fields` behavior was ever changed for special values (None, []),
+        the package should too. https://docs.djangoproject.com/en/models/instances/#specifying-which-fields-to-save
```

Comment:
The link is not working. You can use this one instead: https://docs.djangoproject.com/en/3.2/ref/models/instances/#specifying-which-fields-to-save (3.2 is the latest LTS).